src/pages/admin/Announcements.tsx

```diff
@@ -0,0 +1,960 @@
+              {announcement.publishedDate}
+              <br></br>
+              <b>Expiration Date:</b>
+              {announcement.expirationDate}
+              <br></br>
+              <b>Parish:</b>
+              {announcement.parish}
+            </p>
+          </div>
+        </div>
+      </div>
+    );
+  };
+  /* ============================= List of announcements ============================= */
+  const RenderAnnouncementList = (): JSX.Element => {
+    const announcementsLength: number = announcements?.filter(
+      (a: AnnouncementData) => {
+        if (a.parish === locationFilter) {
+          return true;
+        } else return false;
+      }
+    )?.length;
+    return (
+      <>
+        {announcements
+          .filter((a: AnnouncementData) => {
+            if (a.parish === locationFilter) {
+              return true;
+            } else return false;
+          })
+          .map((announcement: AnnouncementData, index: number) => {
+            if (index < count)
+              return (
+                <RenderAnnouncementItem
+                  key={announcement.id}
+                  announcement={announcement}
+                ></RenderAnnouncementItem>
+              );
+            else return null;
+          })}
+        {announcementsLength === 0 ? <h2>No Announcements found</h2> : null}
+        {announcementsLength &&
+        announcementsLength > 0 &&
+        announcementsLength > count ? (
+          <div
+            style={{
+              textAlign: 'center',
+              marginLeft: 'auto',
+              marginRight: 'auto',
+            }}
+          >
+            <button onClick={() => setCount(count + 5)}>Load More</button>
+          </div>
+        ) : null}
+      </>
+    );
+  };
+
+  const imageHelper = (announcement: AnnouncementData, conditional: string) => {
+    switch (conditional) {
+      case 'true':
+        return `https://themeetinghouse.com/cached/640/static/photos/announcements/${
+          announcement.publishedDate
+        }_${announcement.title.replaceAll(' ', '_')}.jpg`;
+      default:
+        return '';
+    }
+  };
+  /* ============================= Modals ============================= */
+  const EditAnnouncementModal = (): JSX.Element => {
+    const [announcement, setAnnouncement] = useState<AnnouncementData>(
+      currentAnnouncement ?? announcementInit
+    );
+    const [ogAnnouncement] = useState(announcement);
+    const [errorTxt, setErrorTxt] = useState<string>('');
+    const getParishes = () => {
+      const tempArr: Array<string> = [];
+      const ayy = announcements.filter(
+        (a: AnnouncementData) =>
+          a.title === ogAnnouncement.title &&
+          a.publishedDate === ogAnnouncement.publishedDate
+      );
+      for (let i = 0; i < ayy.length; i++) {
+        tempArr.push(ayy[i].parish ?? 'Cross-Regional');
+      }
+      return tempArr;
+    };
+    const [originalParishes] = useState<Array<string>>(getParishes());
```

Comment:
Not sure why this has `useState()`. I think you can just do `const [parishes, setParishes] = useState(getParishes());`